src/many-ledger/src/storage.rs

```diff
@@ -699,6 +743,8 @@ impl LedgerStorage {
             ],
         };
 
+        self.update_data_attributes(from, to, amount.clone(), symbol);
+
```

Comment:
I don't understand your point. This update needs to happen in a specific position of the storage code, and it cannot be generic because other migrations might have hooks in different positions. So it needs to be data-attribute specific.

Reply:
We can improve this, later on, I won't block the merge.